### PR TITLE
SITES-705: Set install_profile variable when one does not exist.

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Set a default profile for anything that isn't dept or product.

# Needed By (Date)
- Shrug

# Urgency
- (1/3)*0.4/10

# Steps to Test

1. Find a site that doesn't have the install_profile var set
2. Migrate using this branch
3. Validate migrated site has stanford as profile.

# Affected Projects or Products
- Migration

# Associated Issues and/or People
- SITES-705

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by sherakama** [comment]
Ok, try this version. It should default out a response to trigger the vset when the variable does not exist.

File: roles/upload-site/tasks/main.yml [hunk 1]
```diff
@@ -61,6 +61,19 @@
     - "-y updb"
   when: ignore_updb_errors == "TRUE"
 
+- name: Get current install profile
+  shell: "{{ drush_alias }} ev 'echo variable_get(\"install_profile\", \"set_me\");'"
```
Comment (jbickar) [inline roles/upload-site/tasks/main.yml]: `drush ev` rather than `drush vget` here, so that you can set a default? SMRT.

**Comment by jbickar** [comment]
This worked on the first pass with `install_profile` unset. Trying again with it set to `stanford_dept`.

File: roles/upload-site/tasks/main.yml (hunk 1 above)
Comment (sherakama) [reply roles/upload-site/tasks/main.yml]: Yeah, I was disappointed that vget did not have a default option.

**Pull request merged**